data_model/src/lib.rs

```diff
@@ -373,6 +379,14 @@ impl From<LengthLimits> for RangeInclusive<usize> {
     }
 }
 
+/// Get the current system time as `Duration` since the unix epoch.
+pub fn current_time() -> Duration {
```

Comment:
```suggestion
pub fn current_time() -> Result<Duration, Error> {
```

Reply:
Originally we have accepted unwrapping by `.except()`.
And I don't think this can fail

Reply:
because the usage is
`SystemTime::now().duration_since(SystemTime::UNIX_EPOCH)`
and `duration_since`
> Returns an `Err` if `earlier` is later than `self`